etc/profile-m-z/qutebrowser.profile

```diff
@@ -27,10 +27,12 @@ include disable-shell.inc
 mkdir ${HOME}/.cache/qutebrowser
 mkdir ${HOME}/.config/qutebrowser
 mkdir ${HOME}/.local/share/qutebrowser
+mkdir ${RUNUSER}/qutebrowser
 whitelist ${DOWNLOADS}
 whitelist ${HOME}/.cache/qutebrowser
 whitelist ${HOME}/.config/qutebrowser
 whitelist ${HOME}/.local/share/qutebrowser
+whitelist ${RUNUSER}/qutebrowser
```

Comment:
Usually a whitelisted path is also added to disable-programs.inc, to block
access to it by default:

```
blacklist ${RUNUSER}/qutebrowser
```

And then access to it would be granted only to programs that need it (like in
qutebrowser.profile):

```
noblacklist ${RUNUSER}/qutebrowser
```

But I'm not sure about sockets.

Is this used just by qutebrowser or also by the programs that try to open tabs?

If you do the above, can the external programs still open tabs?

Reply:
yes, doing the above will keep it working